### DISCUSSION TITLE
use option picker to persist new values

### DISCUSSION THREAD

**Discussion by colonyshipforsale** [Budibase/budibase#6878] [Help] [answered]
Hello all,

is it possible to use the option picker to also add an entry if autocomplete did not yield a result?

Below you can see part of an "add Row" form where the user has to populate several fields and should reuse existing values wherever possible, but add also values, if none where found. Similar to how "tagging" usually works whereas a tag is added if none where found ..?

Worded differently ...

As an "option picker" user i would like to not only autocomplete existing values found within a column (budibase db) (see below) like eg. "doctor" BUT if not found, the same input field that is used for searching, should add a value like eg. "doctor who".

Please let me know if it's possible with reasonable effort? Any help/hint is appreciated, many thanks!

![option picker search and add options](https://user-images.githubusercontent.com/73785468/180738735-a630e1fa-7412-4545-8849-b684ef174359.jpg)

**Comment by melohagan** [comment, marked as answer]
Hello @colonyshipforsale 

I don't think this is currently possible as there is no binding to access the options picker search bar text. 

It's a nice idea though; I've raised a feature request here: https://github.com/Budibase/budibase/issues/6889

**Reply by colonyshipforsale** [reply]
i see, thanks for clearing that up!
